src/Sulu/Bundle/AdminBundle/Resources/js/views/Datagrid/Datagrid.js

```diff
@@ -136,6 +181,14 @@ class Datagrid extends React.Component<ViewProps> {
         router.navigate(addRoute, {locale: this.locale.get(), parentId: rowId});
     };
 
+    handleRequestSelectionDeletion = () => {
```

Comment:
The thing about this method for me is that it is not simply an event handler. For me it is part of the external public API of this component (exactly the same as in `this.datagrid.requestSelectionDelete` below), and can also be called somewhere else, not only in callbacks.

Also, the callback is not registered in this component itself. We can't make a `handle` method for every `ToolbarAction` being added (actually we could, but 3rd party developers can't, so I would try to split this already in our codebase as well...).

Does that make sense?